src/components/InlineCodeBlock/WrappedText.js

```diff
@@ -0,0 +1,76 @@
+import _ from 'underscore';
+import React, {Fragment} from 'react';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,' ',Name,' ',' ',is,' ',Rajat],
+ *  ]
+ *
+ * @param {String} text
+ * @returns {Array<String[]>}
+ */
+function getTextMatrix(text) {
+    return text.split('\n').map(row => _.without(row.split(/(\s)/), ''));
+}
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/19922 when using emojis and text the inline code will look misaligned https://github.com/Expensify/App/issues/19922#issuecomment-1574416955.